src/main/java/org/commoncrawl/spark/CCIndex2Table.java

```diff
@@ -65,6 +65,10 @@ public CdxLine(String line) throws IOException {
 			length = getInt("length");
 			status = getHttpStatus("status");
 
+			crawl = "unknown";
```

Comment:
If we need to support data sets which are not partitioned by crawl, subset and segments, this could be also done by providing a further schema and a adapted table converter, cf. the [EOT schema](https://github.com/commoncrawl/cc-index-table/blob/eot-archive/src/main/resources/schema/eot-index-schema.json) and [EOT converter](https://github.com/commoncrawl/cc-index-table/blob/eot-archive/src/main/java/org/commoncrawl/spark/EOTIndexTable.java).

Reply:
I will try removing this - it was done for testing, and may not be needed now that I mirror the structure of our crawl data during testing.